substratevm/src/com.oracle.svm.truffle/src/com/oracle/svm/truffle/TruffleBaseFeature.java

```diff
@@ -735,22 +741,53 @@ public Object transform(MetaAccessProvider metaAccess, ResolvedJavaField origina
 
 // Checkstyle: stop
 
-// If allowProcess() is disabled at build time, then we ensure that
-// ProcessBuilder is not reachable.
-// The main purpose of this is to test that ProcessBuilder is not part of the
-// image when building
-// language images with allowProcess() disabled, which we interpret as "forbid
-// shelling out to
-// external processes" (GR-14041).
+@TargetClass(className = "com.oracle.truffle.polyglot.PolyglotContextImpl", onlyWith = TruffleBaseFeature.IsEnabled.class)
+final class Target_com_oracle_truffle_polyglot_PolyglotContextImpl {
```

Comment:
That's new, is it really for this PR? It should at least be in its own commit

Reply:
This is a mistake introduced by the rebase (there were conflicts with the master). Should be resolved now.